layouts/calendar/list.html

```diff
@@ -82,4 +82,8 @@ <h1 id="article-title">{{ .Title }}</h1>
         aria-modal="true"
     ></div>
 </main>
+
+{{/*  render the event descriptions as markdown  */}}
+<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
```

Comment:
Let's not load libraries from external sources. Include it in the codebase, please.

Reply:
I did it this way since we don't currently use any package managers. If that's the preferred option I'll refactor.